### PR TITLE
Add support to create cursors.

### PR DESCRIPTION
**Pull request by eiadxp**
This pull request add support to create cursors through XAML extensions. You can change: width, height, fill brush, strike brush and thickness, hot spot point, rotation angle, and flip mode. Default values can be found and changed in the static class MahApps.Metro.IconPacks.PackIconCursorHelper.

Usage:
``` XAML

        <Button xmlns:icons="http://metro.mahapps.com/winfx/xaml/iconpacks"
                Content="{icons:MaterialDesign Kind=AccountBox}"
                Cursor="{icons:MaterialDesignCursor Kind=HourglassEmpty}" />

```

### CODE REVIEW

File: src/MahApps.Metro.IconPacks.Core/PackIconCursorHelper.cs [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+            Rect rect = geometry.Bounds;
+            transformGroup.Children.Add(new TranslateTransform(-rect.X, -rect.Y));
+
+            // Apply the requested size.
+            transformGroup.Children.Add(new ScaleTransform(width / rect.Width, height / rect.Height));
+
+
+            geometry.Freeze();
+            return geometry;
+        }
```
Comment (punker76) [inline src/MahApps.Metro.IconPacks.Core/PackIconCursorHelper.cs:50]: I tested your changes on a sample and get this result. It seems like that the scaling is wrong for the image itself.

2021-12-07_10h03_25

Comment (eiadxp) [reply src/MahApps.Metro.IconPacks.Core/PackIconCursorHelper.cs]: Sorry for the different scaling factor on X,Y dimensions.
I've made changes to correct that.

Best regards

**Comment by punker76** [comment]
@eiadxp Sorry for the delay. I already started to review your last changes. But I still need some time for finishing it. I will try to do this in the next 2 weeks. ❤️🙏🏻

**Comment by eiadxp** [comment]
> @eiadxp Sorry for the delay. I already started to review your last changes. But I still need some time for finishing it. I will try to do this in the next 2 weeks. ❤️🙏🏻

Thanks for all your efforts 🙂

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/MahApps/MahApps.Metro.IconPacks?pullRequest=284) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/MahApps/MahApps.Metro.IconPacks?pullRequest=284) before we can accept your contribution.<br/><br/>:x: eiadxp [sign now](https://cla.dotnetfoundation.org/MahApps/MahApps.Metro.IconPacks?pullRequest=284)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/MahApps/MahApps.Metro.IconPacks?pullRequest=284) it.</sub>

**Review by punker76 — APPROVED** [review]
LGTM 👍

**Pull request merged**